examples/conf/druid/cluster/data/indexer/jvm.config

```diff
@@ -0,0 +1,9 @@
+-server
+-Xms128m
+-Xmx128m
+-XX:MaxDirectMemorySize=1g
```

Comment:
we probably need more heap to match our middle-manager config, which had 1GB per task.

Reply:
Made it 4g and also increased `druid.processing.buffer.sizeBytes` to 500MiB